Admin/index.html

```diff
@@ -273,16 +275,17 @@ <h6 class="card-subtitle">Website Sales</h6>
             <div class="col-sm-6 col-lg-3 column-divider-lg">
               <div class="media">
                 <div class="media-body">
-                  <h6 class="card-subtitle">Discount</h6>
-                  <span class="card-title h3">$15,503.00</span>
-                
+                  <h6 class="card-subtitle">Total Sales</h6>
+                  <span class="card-title h3" id="total-sales">$15,503.00</span>
```

Comment:
[Optional]
Ideally we should use camel casing for all IDs to maintain consistency.
If you've time, please update the IDs which you've created to camel casing